01.w_Defects/invalid_memory_access.c

```diff
@@ -30,7 +30,7 @@ void invalid_memory_access_001 ()
 	int *ptr, a;
 	int flag=10;
 
-    (flag == 10)? (ptr= (int*) malloc(10*sizeof(int))) : ( a= 5);
+    (flag == 10)? (void)(ptr= (int*) malloc(10*sizeof(int))) : (void)( a= 5);
```

Comment:
this is a constraint violation. The type of the first branch is int * and the second is int which it is not compatible with. This violates C11 sec. 6.5.15 paragraph 3.